server/command.go

```diff
@@ -333,9 +333,16 @@ func (p *Plugin) executeShowLinksCommand(args *model.CommandArgs) (*model.Comman
 	}
 
 	p.sendBotEphemeralPost(args.UserId, args.ChannelId, commandWaitingMessage)
-	recovery.Go("send_links_with_details", p.API.LogError, func() {
+	go func() {
+		defer func() {
```

Comment:
This could be moved inside the `SendLinksWithDetails` function.

Reply:
This one makes sense -- pushed! -- as the goroutine and implementation are all co-located.